au4/src/projectscene/qml/Audacity/ProjectScene/clipsview/TrackClipsItem.qml

```diff
@@ -197,11 +205,28 @@ Item {
         }
     }
 
+    ChannelSplitter {
+        id: channelSplitter
+
+        anchors.fill: parent
+        anchors.topMargin: trackViewState.isTrackCollapsed ? 1 : 21
+        anchors.bottomMargin: 3
+
+        channelHeightRatio: root.channelHeightRatio
+        color: "#FFFFFF"
```

Comment:
can we use some value from theme instead of hardcode here?

Reply:
Not sure, lets ask @DilsonsPickles

Reply:
Hey, can I get a reference so I know which part of the design we're talking about?

Reply:
@DilsonsPickles, here we are talking about the stereo channel separator. 
@Eism, to clarify your question, are you talking about using an already-defined color from the theme or introducing a new one? From @DilsonsPickle's answers, I understand that the colors for the channel separator are theme-independent.

Reply:
@kryksyh colours are the same across both themes for both the channel separator that is on the clip and canvas. 

To reiterate values:

Clip channel separator - Black-color @ 10% opacity
Canvas channel separator - White-colour @ 5% opacity